app/src/components/point_cloud_canvas.tsx

```diff
@@ -88,7 +96,7 @@ const fragmentShader = `
 
     void main() {
       float alpha = 0.5;
-      vec3 color = getHeatMapColor(worldPosition.z);
+      vec3 color = pointColor;
```

Comment:
In the original point cloud labeling, the points that are inside a box would be highlighed as RED. We should keep this behavior.